timev.rb

```diff
@@ -1,112 +1,124 @@
 #
 # call-seq:
-#    Time.now -> time
+#    Time.now -> new_time
 #
-# Creates a new Time object for the current time.
+# Creates a new \Time object from the current system time.
 # This is the same as Time.new without arguments.
 #
-#    Time.now            #=> 2009-06-24 12:39:54 +0900
+#    Time.now # => 2009-06-24 12:39:54 +0900
 def Time.now(in: nil)
   new(in: __builtin.arg!(:in))
 end
 
 #
 # call-seq:
```

Comment:
I think we should remove call-seq line for methods defined in Ruby.  Similar issue below.

Reply:
Do you mean to remove call-seq for Time.now, Time.at, and Time.new? And that would let the call-seqs fall through to RDoc's default?

Reply:
Yes.  call-seq should only be used for methods defined in C.  For methods defined in Ruby, RDoc's default should be used.

Reply:
Done, but note that there are no return values for these.